pkg/terraform/executor.go

```diff
@@ -2,59 +2,20 @@ package terraform
 
 import (
 	"bytes"
-	"os"
 	"os/exec"
 	"path/filepath"
-	"runtime"
-	"strings"
 
 	"github.com/openshift/installer/pkg/lineprinter"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
-// executor enables calling Terraform from Go, across platforms, with any
-// additional providers/provisioners that the currently executing binary
-// exposes.
-//
-// The Terraform binary is expected to be in the executing binary's folder, in
-// the current working directory or in the PATH.
-type executor struct {
-	binaryPath string
-}
-
-// Set the binary names for different platforms
-const (
-	tfBinUnix    = "terraform"
-	tfBinWindows = "terraform.exe"
```

Comment:
Hmm.  I wonder if we need `.exe` on Windows?

Reply:
Probably, but we aren't going to support Windows initially. I figured it was easier to simplify this logic than to try to fix how this would work on Windows.